native/src/boot/bootimg.cpp

```diff
@@ -895,7 +963,7 @@ void repack(const char *src_img, const char *out_img, bool skip_comp) {
 }
 
 int verify(const char *image, const char *cert) {
-    const boot_img boot(image);
+    const boot_img boot(image, true);
```

Comment:
doubt if it should be hardcoded `true`

Reply:
In my opinion, this is fine because it assures the same behavior as before and has no impact on extracted files.